packages/playground/src/index.css

```
@@ -86,3 +86,75 @@ th:has(header) {
+  .starter-aurora {
+    position: relative;
+    overflow: hidden;
+    isolation: isolate;
+  }
+
+  .starter-aurora::before {
+    content: '';
+    position: absolute;
+    inset: -20%;
+    background:
+      radial-gradient(ellipse 55% 45% at 25% 30%, rgba(212, 212, 216, 0.09), transparent 65%),
```

Stylelint (17.12.0)

[error] Expected "0.09" to be "9%" (alpha-value-notation)

(alpha-value-notation)